src/test/e2e/14_oci_compose_test.go

```diff
@@ -176,6 +174,12 @@ func (suite *PublishCopySkeletonSuite) Test_2_FilePaths() {
 			isSkeleton = true
 		}
 		suite.verifyComponentPaths(unpacked, components, isSkeleton)
+
+		// Cleanup resources
+		err = os.RemoveAll(unpacked)
```

Comment:
IMO we should keep these as defers so that if there is a failure during the test the resources still get cleaned up. We can do this to check the error
```go 
		defer func(){
			suite.NoError(os.RemoveAll(unpacked))
		}()
```

Reply:
Thank you for catching that, I don't remember what my justification was for moving those cleanups out of defers.